automation/check_artifact_size.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# A simple check that our release .aar files are a reasonable size.
+# If this fails then something has gone wrong with the build process,
+# such as pulling in unwanted dependencies or failing to strip debug symbols.
+
+set -eu
+
+if [ "$#" -ne 2 ]
+then
+    echo "Usage:"
+    echo "./automation/check_artifact_size.sh <buildDir> <artifactId>"
+    exit 1
+fi
+
+BUILD_DIR="$1"
+ARTIFACT_ID="$2"
+
+# Even our largest .aar should be less than 30M.
+# Seems like a lot? They include compiled rust code for 4 architectures.
+# We expect this size to decrease over time as we make changes to the way
+# we perform megazord builds, but at least it's an upper bound for now..
+LIMIT=30000000
```

Comment:
I settled on this initial limit by doing a local build of our largest components:

```
$ find ~/.m2/repository/org/mozilla/appservices/ -name "*.aar" | xargs du -hs
12M     /home/rfk/.m2/repository/org/mozilla/appservices/fenix-megazord/0.27.2/fenix-megazord-0.27.2.aar
17M     /home/rfk/.m2/repository/org/mozilla/appservices/lockbox-megazord/0.27.2/lockbox-megazord-0.27.2.aar
27M     /home/rfk/.m2/repository/org/mozilla/appservices/places/0.27.2/places-0.27.2.aar
```

By contrast, the current [places-0.27.1.aar](https://maven.mozilla.org/maven2/org/mozilla/appservices/places/0.27.1/places-0.27.1.aar) weighs in at over 80M.

Reply:
I wonder if we could add a simple script to one of our webpages that retrieves our published artifacts on `maven.mozilla.org` and plots a graph of our aar sizes, so we can have a general idea of our size changes.